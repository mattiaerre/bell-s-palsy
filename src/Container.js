import useLocalStorage from './hooks/useLocalStorage';
import App from './App';
import paths from './files.json';
import Playground from './Playground';

function Container() {
  const [authorization, setAuthorization] = useLocalStorage(
    'bell-s-palsy',
    'unauthorized'
  );

  if (authorization !== 'authorized') {
    return <Playground callback={setAuthorization} />;
  }
  return <App callback={setAuthorization} paths={paths} version="0.7.5" />;
}

export default Container;
