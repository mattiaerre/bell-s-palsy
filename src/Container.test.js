import { render } from '@testing-library/react';
import Container from './Container';

test('snapshot', () => {
  const { asFragment } = render(<Container />);
  expect(asFragment()).toMatchSnapshot();
});
