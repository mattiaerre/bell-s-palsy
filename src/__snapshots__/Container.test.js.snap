// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`snapshot 1`] = `
<DocumentFragment>
  <div
    class="App"
  >
    <button
      class="Close"
      disabled=""
    >
      Ã—
    </button>
    <h1>
      Bell's palsy - Mattia's journey
    </h1>
    <section
      class="Playground"
    >
      <input
        class="Number"
        maxlength="1"
        placeholder="b"
        type="text"
      />
      <input
        class="Number"
        maxlength="1"
        placeholder="e"
        type="text"
      />
      <input
        class="Number"
        maxlength="1"
        placeholder="l"
        type="text"
      />
      <input
        class="Number"
        maxlength="1"
        placeholder="l"
        type="text"
      />
    </section>
    <footer>
      bell-s-palsy v0.9.0 - 2022
    </footer>
  </div>
</DocumentFragment>
`;
